Modules/unicodedata.c

```diff
@@ -1477,13 +1491,20 @@ unicodedata_exec(PyObject *module)
     }
 
     /* Export C API */
```

Comment:
Maybe you can move this code into unicodedata_create_capi() function?

Reply:
Yes, that's an improvement.